Logger/BaseLogger.cs

```diff
@@ -2,6 +2,8 @@
 {
     public abstract class BaseLogger
     {
+        public string className { get; set; } 
```

Comment:
Properties should be named with PascalCase.
```suggestion
        public string ClassName { get; set; } 
```